SPECS/jx/jx.spec

```
@@ -36,10 +37,12 @@ BuildRequires:  golang >= 1.17.1
 Command line tool for working with Jenkins X.
 
 %prep
-%autosetup -p1
+%autosetup -N -p1
```

nit: With the `-N` option, it won't apply patches, which renders `-p1` moot.
```suggestion
%autosetup -N
```